### ISSUE TITLE
When pre-loading reviews, create only the number of reviews that the page will load

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#2664]
### Describe the problem and steps to reproduce it:
Before the reviews on the "All reviews" page have loaded, we render a few reviews with `<LoadingText />`. But if we're coming from the add-on details page, we have a review count in state already so we should render that many reviews. This prevents an add-on with only, say, one review from having four "loading" reviews and then cutting the content down to only one review once loaded.

### Anything else we should know?
If you access the page directly, obviously we won't have a review count. In that case we can continue to load a random/set amount.

**Comment by willdurand** [comment]
@kumar303 given all the work around reviews, is this issue still relevant?

**Comment by willdurand** [comment]
Thanks! That'd be a nice improvement indeed

**Comment by kumar303** [comment]
Yep. It should be doable although I haven't looked at it closely.

**Comment by SeanPrashad** [comment]
Hey guys, I'm gunna take a crack at this. The source code in question is found here:

https://github.com/mozilla/addons-frontend/blob/fda47b4842a7066ce989a7e5a6ec1a84ecd479fc/src/amo/pages/AddonReviewList/index.js#L232-L240

**Comment by SeanPrashad** [comment]
> It still happens; coming from the main add-ons page we know how many reviews there are, so it would make sense to show that many `LoadingText` reviews. This one, for instance, should have 28 and not 4:
> 
> ![2018-11-13 09 22 14](https://user-images.githubusercontent.com/90871/48403454-c9306b80-e725-11e8-89c8-fc6ecf6ee672.gif)
> 
> It's pretty minor in that case, but it's a bit more jarring when there are _fewer_ than 4 reviews:
> 
> ![2018-11-13 09 24 16](https://user-images.githubusercontent.com/90871/48403515-ea915780-e725-11e8-8f4e-32d92a2646e6.gif)

@tofumatt Would there be **25** or **28** `LoadingText` components? The reason being is that I thought pagination only allows 25 to show at a time - please correct me if I'm wrong

**Comment by tofumatt** [comment]
I'd say the limit should be the pagination limit; I wouldn't show 28 when the most that would appear would be 25 👍

**Comment by SeanPrashad** [comment]
Great!! Thanks for the super speedy reply 🚀

**Comment by ioanarusiczki** [comment]
@SeanPrashad @willdurand I verified this issue on AMO dev (FF63, Win10).
To see this behavior I first checked on AMO prod this behavior and I could indeed reproduce it but only for add-ons with more than 25 reviews, never with 4 reviews or below, and also it is intermittent. 
I could reproduce it on prod with
https://addons.mozilla.org/en-US/firefox/addon/privacy-possum/
https://addons.mozilla.org/en-US/firefox/addon/steam-database/
https://addons.mozilla.org/en-US/firefox/addon/arc-dark-theme-we/

I was not able to reproduce it on AMO dev at all. This should no longer be an issue but to be sure I'll recheck it with the above samples on AMO prod too.
